shared/js/package.json

```diff
@@ -3,26 +3,26 @@
   "version": "0.0.1",
   "description": "Code.org shared JS code",
   "engines": {
-    "node": ">=0.10.25 <0.13",
+    "node": ">=0.12.0 <0.13",
     "npm": "^2.9.1"
   },
   "scripts": {
-    "gulp": "gulp",
-    "build": "gulp -- bundle-js",
-    "clean": "gulp -- clean",
-    "lint": "gulp -- lint",
-    "dev": "gulp -- watch"
+    "build": "npm run build-js",
+    "build-js": "./build-js.js",
+    "build-js:min": "./build-js.js --min",
```

Comment:
We could. I liked the symmetry with code-studio. If we move this code into code-studio (which I think we should) it becomes a non-factor.